### DISCUSSION TITLE
How is this related to Fork?

### DISCUSSION THREAD

**Discussion by varijkapil13** [Uncommon/Xit#439] [Q&A] [answered]
Hi guys,
I was trying to find how to contribute to Fork, because I come from GitKraken and really liked some of the features they have in there. I found Xit in the process and I must say, Xit and Fork look a lot alike. Fork on the other hand have different styles and bit more features, but the way lot of things are setup, one might wonder, if they share more than just design choices?

In any case, will you guys be willing to accept PRs regarding usability/design as well as functionality? I know a few developers who have tried both GitKraken and Fork and they and me included, like a few features from both of them that I would be willing to contribute here.

Cheers!

**Comment by Uncommon** [comment, marked as answer]
Well I think the answer to the first question is no, the relationship between Xit and Fork is nothing more than superficial.

The other answer, of course, is yes, I'm totally open to input on the user interface.

**Comment by Uncommon** [comment]
Before you start working on a pull request I'd recommend you file an issue about the change you want to make, so we can discuss it and make sure we don't duplicate our efforts :)

**Comment by varijkapil13** [comment]
Oh great! Thanks I will open a new new issue for some changes to the UI
